src/main/java/io/fabric8/maven/docker/util/EnvUtil.java

```diff
@@ -308,7 +308,7 @@ public static String findRegistry(String ... checkFirst) {
     }
 
     public static File prepareAbsoluteOutputDirPath(MojoParameters params, String dir, String path) {
-        return prepareAbsolutePath(params, new File(params.getOutputDirectory(), dir).toString(), path);
+        return prepareAbsolutePath(params, new File(new File(params.getProject().getBuild().getDirectory(), params.getOutputDirectory()), dir).toString(), path);
```

Comment:
Don't really get this line: If `params.getOutputDirectory()` is already an absolute path (it think, this is what this PR is about to support) why would I want it to append it to another directory?